### ISSUE TITLE
System Error when running Ganache 2.0.0 on darwin

### ISSUE THREAD

**Issue by Agus** [trufflesuite/ganache#1233]
PLATFORM: darwin
GANACHE VERSION: 2.0.0

EXCEPTION:
```
TypeError: Cannot read property 'nodes' of undefined
    at getStateVariables (/node_modules/truffle-decoder/dist/allocate/storage.js:146:24)
    at vars.concat.linearizedBaseContractsFromBase.map (/node_modules/truffle-decoder/dist/allocate/storage.js:154:73)
    at Array.map (native)
    at allocateContract (/node_modules/truffle-decoder/dist/allocate/storage.js:154:61)
    at Object.getStorageAllocations (/node_modules/truffle-decoder/dist/allocate/storage.js:28:23)
    at TruffleContractDecoder.init (/node_modules/truffle-decoder/dist/interface/contract-decoder.js:87:43)
    at Object.getContractState (/src/truffle-integration/decode.js:18:11)
    at process.<anonymous> (/src/truffle-integration/index.js:76:31)
    at emitTwo (events.js:125:13)
    at process.emit (events.js:213:7)
    at emit (internal/child_process.js:768:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+123663ms: eth_newBlockFilter
T+123747ms: eth_getFilterChanges
T+123829ms: eth_getTransactionReceipt
T+123911ms: eth_getCode
T+123997ms: eth_uninstallFilter
T+124074ms: eth_sendTransaction
T+124149ms:   Transaction: 0x26cf78498ef45ef7fc1b6e1ccaca5ec0dbdfe5f7e605205a4c81171021dae395
T+124149ms:   Gas usage: 26908
T+124236ms:   Block Number: 82
T+124236ms:   Block Time: Wed Apr 10 2019 17:12:54 GMT-0300 (-03)
T+124315ms: eth_getTransactionReceipt
T+124395ms: eth_accounts
T+124457ms: net_version
T+124531ms: net_version
T+124606ms: eth_sendTransaction
T+124679ms:   Transaction: 0xd8a31325c2c84bfb6de9619e55d2b6adbd6ad2c9f4767e8c64b534cb5ba7e1a6
T+124679ms:   Contract created: 0x123aa54f7726b3b642133995a9e29ba1a5eecef5
T+124679ms:   Gas usage: 227624
T+124801ms:   Block Number: 83
T+124801ms:   Block Time: Wed Apr 10 2019 17:12:54 GMT-0300 (-03)
T+124874ms: eth_newBlockFilter
T+124945ms: eth_getFilterChanges
T+125005ms: eth_getTransactionReceipt
T+125073ms: eth_getCode
T+125145ms: eth_uninstallFilter
T+125215ms: net_version
T+125275ms: net_version
T+125354ms: eth_sendTransaction
T+125427ms:   Transaction: 0x20f9dbad7e21d943bdb66db288b62d0e34a3a464ba8006a37dba4526fa761fb3
T+125427ms:   Contract created: 0xca35c8620b4b77e2621cd4072b50e9efdf7a03a0
T+125427ms:   Gas usage: 2040096
T+125427ms:   Block Number: 84
T+125485ms:   Block Time: Wed Apr 10 2019 17:12:55 GMT-0300 (-03)
T+125553ms: eth_newBlockFilter
T+125621ms: eth_getFilterChanges
T+125689ms: eth_getTransactionReceipt
T+125746ms: eth_getCode
T+125828ms: eth_uninstallFilter
T+125896ms: eth_sendTransaction
T+125966ms:   Transaction: 0xa7473ba9eb0dadf89e4319e693fd592ddebd703f2d8fbbefdcae492d68f223d7
T+125966ms:   Contract created: 0xdea07138a183bbf8a4fa02544e61fae877f87747
T+125966ms:   Gas usage: 3240196
T+126029ms:   Block Number: 85
T+126029ms:   Block Time: Wed Apr 10 2019 17:12:55 GMT-0300 (-03)
T+126106ms: eth_newBlockFilter
T+126189ms: eth_getFilterChanges
T+126257ms: eth_getTransactionReceipt
T+126331ms: eth_getCode
T+126405ms: eth_uninstallFilter
T+126484ms: eth_sendTransaction
T+126542ms:   Transaction: 0x1fc56d73103535b03ee5c76ccc16f30a64af945bb718a2bc67fe7fe2e2d2c5f6
T+126542ms:   Gas usage: 26908
T+126542ms:   Block Number: 86
T+126611ms:   Block Time: Wed Apr 10 2019 17:12:55 GMT-0300 (-03)
T+126679ms: eth_getTransactionReceipt
T+126751ms: eth_accounts
T+126815ms: eth_sendTransaction
T+126902ms:   Transaction: 0x5c0cc4217b1fc72d6c0935e0156f8ecd0ec15ad9caece5556d0416f306c8ce94
T+126902ms:   Contract created: 0xb36db9c55cb5d70a9095b32e37a889bb3cbc5280
T+126902ms:   Gas usage: 5065290
T+127090ms:   Block Number: 87
T+127090ms:   Block Time: Wed Apr 10 2019 17:12:56 GMT-0300 (-03)
T+127300ms: eth_newBlockFilter
T+127415ms: eth_getFilterChanges
T+127537ms: eth_getTransactionReceipt
T+127647ms: eth_getCode
T+127736ms: eth_uninstallFilter
T+127853ms: eth_sendTransaction
T+127964ms:   Transaction: 0x9b578259725224a01f8057f8477cf4ba785c76411f2adb94598cd49bc694961a
T+127964ms:   Gas usage: 26908
T+127964ms:   Block Number: 88
T+128083ms:   Block Time: Wed Apr 10 2019 17:12:56 GMT-0300 (-03)
T+128200ms: eth_getTransactionReceipt
T+128272ms: eth_accounts
T+128353ms: eth_getBlockByNumber
T+128436ms: eth_getBlockByNumber
T+128660ms: eth_sendTransaction
T+128741ms:   Transaction: 0xc8b4e354db7bf04f26e5a3feef093d7566b68bc5a6a89ed23209826b18d8cd86
T+128741ms:   Contract created: 0x1c7d5555f6317cef175c789a4401472c39a12136
T+128741ms:   Gas usage: 1011456
T+128741ms:   Block Number: 89
T+128811ms:   Block Time: Wed Apr 10 2019 17:12:56 GMT-0300 (-03)
T+128874ms: eth_newBlockFilter
T+128941ms: eth_getFilterChanges
T+129028ms: eth_getTransactionReceipt
T+129438ms: eth_getCode
T+129733ms: eth_uninstallFilter
T+129798ms: eth_sendTransaction
T+129887ms:   Transaction: 0x132f4b88c44062b0ac5bc7c8ed9b7050d50569a6a7bd65ea99b2bbbd91906b58
T+129887ms:   Gas usage: 26908
T+129887ms:   Block Number: 90
T+130021ms:   Block Time: Wed Apr 10 2019 17:12:57 GMT-0300 (-03)
T+130221ms: eth_getTransactionReceipt
T+130295ms: eth_getBlockByNumber
T+130369ms: eth_getBlockByNumber
T+130451ms: eth_getBlockByNumber
T+130556ms: eth_getBlockByNumber
T+130670ms: eth_getBlockByNumber
T+130769ms: eth_getBlockByNumber
T+130867ms: eth_getBlockByNumber
T+130940ms: eth_getBlockByNumber
T+131016ms: eth_getBlockByNumber
T+131087ms: eth_getBlockByNumber
T+150163ms: eth_unsubscribe
T+150240ms: eth_unsubscribe
T+150309ms: eth_unsubscribe
T+150309ms: eth_unsubscribe
```

**Comment by davidmurdoch** [comment]
Hey @Agus, thanks for opening this issue. This bug should now be fixed in our latest beta (https://github.com/trufflesuite/ganache/releases/tag/v2.0.1-beta.0). Feel free to try out the beta now if you can't wait for the fix to land in v2.0.1 (in about a week from now).

**Comment by davidmurdoch** [comment]
_Now_ this issue should be fixed in the latest beta! https://github.com/trufflesuite/ganache/releases/tag/v2.0.2-beta.0
